packages/react-fs/src/ReactFilesystem.js

```diff
@@ -20,19 +20,19 @@ const Rejected = 2;
 type PendingRecord = {|
   status: 0,
   value: Wakeable,
-  cache: Array<mixed>,
+  cache: null,
```

Comment:
All of these encodings default to "utf8" which is the common case. So might make more sense to eagerly allocate these.

Reply:
Note I don't end up using this field for most. It's actually only used for `readFile`.

This is because the rest need a Promise-per-cache-entry. So it's the other way around for them. `Map { path => [mixed, Record, mixed, Record, ...] }`